TEAM ATSのDevelopです。5/30 時点の状況です。

Kochinda opened the pull request.

Kochinda committed:
initial test commite

Commit:
test a

Kochinda committed:
Merge pull request #1 from Kochinda/asako

test a

hashimotobsk24 committed:
Merge pull request #2 from Kochinda/sayaka

initial test commite test

hashimotobsk24 committed:
test about

asako05 committed:
Merge pull request #3 from Kochinda/takanori

test about

Kochinda committed:
add htim and home-bg img

Kochinda committed:
add counfdown timer

Commit:
works.編集しました。

Kochinda committed:
途中経過

Kochinda committed:
アニメーション追加

Kochinda committed:
ツアーのロゴ追加、カウントダウンタイマーレイアウト変更

Kochinda committed:
隠しボタン追加

hashimotobsk24 committed:
Merge pull request #4 from Kochinda/sayaka

Sayaka

hashimotobsk24 committed:
5/30 10:00

Kochinda committed:
Merge pull request #5 from Kochinda/asako

works.編集しました。

Kochinda committed:
Merge pull request #6 from Kochinda/takanori

5/30 10:00

hashimotobsk24 committed:
イメージかえた

hashimotobsk24 committed:
Merge pull request #7 from Kochinda/takanori

イメージかえた

Kochinda committed:
歌追加

Commit:
Asa img追加

Kochinda committed:
Merge pull request #8 from Kochinda/asako

Asa img追加

Kochinda committed:
Header navi 追加

Kochinda committed:
音楽追加 Merge branch 'develop' of https://github.com/Kochinda/ATS-project into develop

Ai0202 commented on sass/pages/home.scss:
```diff
@@ -0,0 +1,160 @@
+// _と拡張子を忘れずに！**でファイル中身を全部コピーできる。
+// @import "../base/_reset.scss";
```
ごめんなさい。sassは拡張子なくてもimportできます。
昨日はGulpのエラーでSass記法が使えませんでした。

Ai0202 commented on sass/pages/home.scss:
```diff
@@ -0,0 +1,160 @@
+}
+
+@mixin animation($animation-name) {
+  -webkit-animation: $animation-name;
```
ベンダープレフィックスはつけなくてもコンパイルした時に勝手にくっつきます。

Ai0202 commented on sass/pages/home.scss:
```diff
@@ -0,0 +1,160 @@
+%posion{
+    z-index: 3;
+    position: absolute;
+} 
```
👍

hashimotobsk24 committed:
背景、プロフィール画面作成

Commit:
works完成？！

hashimotobsk24 committed:
Merge pull request #9 from Kochinda/asako

works完成？！

Kochinda committed:
Merge pull request #10 from Kochinda/takanori

背景、プロフィール画面作成。できたら追加でフッター、アニメーション作ってみます。

Kochinda committed:
ホーム完成！

Kochinda committed:
嵐の写真。私が間違えて何かしたかも？

Kochinda committed:
さやか Merge branch 'develop' of https://github.com/Kochinda/ATS-project into develop

Kochinda committed:
コンフリクト修正済み
Merge branch 'develop' into sayaka

# Conflicts:
#	sass/base/_base.scss
#	sass/pages/about.scss

Kochinda committed:
Merge pull request #11 from Kochinda/sayaka

Sayaka Home完成だけど、マージ怪しい。。

Kochinda committed:
Almost!完成。問題：タカのhtmlに貼ったWorks.js が読み込まれてないかもしれない。

hashimotobsk24 committed:
Merge pull request #13 from Kochinda/sayaka

Almost!完成。問題：タカのhtmlに貼ったWorks.js が読み込まれてないかもしれない。

hashimotobsk24 committed:
no message

hashimotobsk24 committed:
Merge branch 'develop' into takanori

# Conflicts:
#	about.html
#	sass/style.scss

hashimotobsk24 committed:
完成？

Kochinda committed:
Merge pull request #14 from Kochinda/takanori

Takanori 完成？？

hashimotobsk24 committed:
300

Kochinda committed:
Merge pull request #15 from Kochinda/takanori

300

Kochinda commented:
完成です！
